packages/preview-middleware-client/src/cpe/outline/nodes.ts

```diff
@@ -136,15 +141,41 @@ export async function transformNodes(input: OutlineViewNode[], scenario: Scenari
     return items;
 }
 
+/**
+ * Fill reuse components ids.
+ *
+ * @param reuseComponentsIds ids of reuse components that are filled when outline nodes are transformed
+ * @param node view node
+ * @param scenario type of project
+ */
+async function fillReuseComponents(
+    reuseComponentsIds: Set<string>,
+    node: OutlineViewNode,
+    scenario: Scenario
+): Promise<void> {
+    const version = sap.ui.version;
+    const minor = parseInt(version.split('.')[1], 10);
+    let Component = (await import('sap/ui/core/Component')).default;
```

Comment:
Why not directly import it at the top of the file `import Component from "sap/ui/core/Component"`? Is it because it is version dependent, then probably the import should be moved one line below, below the check for version?

Should the Component be a const if it is not reassigned later?

Reply:
Only the method that is used by from the `Component` is version dependent. Fixed in #[9db3adc](https://github.com/SAP/open-ux-tools/pull/1941/commits/9db3adc6d45fb48e65b90c7939b16ef24d0a7163) and #[82df8e7](https://github.com/SAP/open-ux-tools/pull/1941/commits/82df8e7f4ceefdacbf7eee18725039f206c373b3)